ORE_EEG_2.py

```diff
@@ -185,7 +342,7 @@ def run_experiment(trials, window, subj, session_number, n_images, all_images):
 
             # Display break message at the end of each block
             display_break_message(window, trial['block'])
-
+    await teardown_eeg()
```

Comment:
why do we call this twice?

Reply:
whoops, that was remenants from testing, i don't think it ever reached the call on line 421 when i was testing it but hopefully that's just because i was exiting the experiment early/no actual heardware

Reply:
no worries! Just remote it pls